Sources/MessageData.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ MIT License
+ 
+ Copyright (c) 2017 MessageKit
+ 
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+ 
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+ 
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+import Foundation
+import class CoreLocation.CLLocation
+
+public enum MessageData {
+    
+    case text(String)
+    
+    case attributedText(NSAttributedString)
+    
+    case audio(Data)
+    
+    case location(CLLocation)
+    
+    case photo(UIImage)
+    
+    case video(file: NSURL, thumbnail: UIImage)
+    
+    case system(String)
```

Comment:
I'm actually not sure because this is code leftover from the old repo. The only thing to be concerned about for the moment is `MessageData.text`. v1 should be basic messages and then expand to Media data types from there.

@towry eventually the code will be fully commented by v1. right now I'm just hacking things together 😅